src/rating/__tests__/rating.test.js

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import * as React from 'react';
+import {shallow} from 'enzyme';
+import {Rating} from '../';
+
+describe('Rating', () => {
+  let example, onChangeSpy;
+
+  beforeEach(() => {
+    onChangeSpy = jest.fn();
+    example = shallow(<Rating value={2} onChange={onChangeSpy} />);
+  });
+
+  describe('Root', () => {
+    it('applies correct accessibility attributes to the Root element', () => {
+      expect(example).toHaveProp('tabIndex', 0);
+      expect(example).toHaveProp('role', 'radiogroup');
+    });
+  });
+
+  describe('RatingItem', () => {
+    it('applies correct props if item is active', () => {
+      expect(example.childAt(0)).toMatchSnapshot();
```

Comment:
`childAt` is not semantically readable. Should we use `find` to look for exact element?

Reply:
Due to shallow rendering and all of the children being the same element type; I'm not sure how find would be more readable.  Do you have a specific example you had in mind?

Reply:
https://airbnb.io/enzyme/docs/api/ShallowWrapper/at.html